source/userguide/active_learning.rst

```diff
@@ -182,7 +193,7 @@ The tuning step allows the application to run 7 possible strategies (``tuning_st
 +---------------------------+-----------------------------------------------------------------------------------------------+
 | Disagreement Sampling     | Across n runs of the classifier, this sampling strategy calculates an agreement score for     |
 |                           | every query (% of classifiers that voted for the most frequent class). The queries are then   |
-|                           | ranked from lowest classifier agreement to highest and then sampled in order.                  |
+|                           | ranked from lowest classifier agreement to highest and then sampled in order.                 |
```

Comment:
To clarify, a lot of these line changes look identical visually. Is the change here related to spaces/tabs?